node_modules/gh-auth/tests/util.js

```diff
@@ -882,11 +886,13 @@ var assertOauthAccessToken = module.exports.assertOauthAccessToken = TestsUtil.c
                     });
                 },
                 'updatedAt': function(actualUpdatedAt) {
+                    // TODO: Enable this when https://github.com/sequelize/sequelize/pull/3733 is resolved
```

Comment:
It seems like the sequelize guys moved faster than we did. Sequelize 3.1.1 has been released and contains that PR.